internal/beater/beater.go

```diff
@@ -828,6 +828,7 @@ func (s *Runner) newLibbeatFinalBatchProcessor(
 			// No output defined, so the acker will never be closed.
 			return nil
 		}
+		defer acker.ClientClosed()
```

Comment:
Should we be calling this directly? Looking at libbeat code, it appears to be called by when the pipeline.Client is closed.

Reply:
Good catch! Fixed!